build.gradle

```diff
@@ -15,21 +15,23 @@ apply plugin: 'eclipse'
 apply plugin: 'org.springframework.boot'
 
 group = 'com.greenfoxacademy.opal.kalendaryo'
-version = '0.0.3-SNAPSHOT'
+version = '0.0.4-SNAPSHOT'
 sourceCompatibility = 1.8
 
 repositories {
 	mavenCentral()
 }
 
-
 dependencies {
 	compile('org.springframework.boot:spring-boot-starter-data-jpa')
 	compile('org.springframework.boot:spring-boot-starter-web')
 	compile('com.google.apis:google-api-services-calendar:v3-rev271-1.23.0')
 	runtime('org.springframework.boot:spring-boot-devtools')
 	runtime('mysql:mysql-connector-java')
 	compileOnly('org.projectlombok:lombok')
+	compile group: 'com.squareup.retrofit2', name: 'retrofit', version: '2.3.0'
```

Comment:
styling, all dependencies should look the same

Reply:
we don't need them actually